projects/kit/components/input-month-range/input-month-range.template.html

```diff
@@ -12,7 +12,7 @@
         [disabled]="computedDisabled"
         [nativeId]="nativeId"
         [readOnly]="readOnly"
-        [tuiTextfieldIcon]="calendarIcon"
+        [tuiTextfieldIcon]="calendarIcon ? calendarIcon : null"
```

Comment:
what about :))

```html
[tuiTextfieldIcon]="calendarIcon | tuiFallback: null"
```

where

```ts
export class TuiFallbackPipe {
  transform<T, K = T>(value?: T|null, fallback: K): T | K {
    return tuiIsPresent(value) ? value : fallback;
  } 
}
```

Reply:
I don't mind, but I want to here @waterplea's opinion on this one, since it's our favourite topic in this PR 😄

Reply:
but again, it doesn't cover a [case](https://github.com/Tinkoff/taiga-ui/pull/2966#discussion_r1009408564) with an empty string

Reply:
Oh, sorry, you can use simple solution

```

```html
[tuiTextfieldIcon]="calendarIcon ?? null"
```
```

Reply:
oh right, I didn't even think about it for some reason